src/main/java/com/spacesongr/songr/Controllers.java

```diff
@@ -0,0 +1,35 @@
+package com.spacesongr.songr;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import java.util.ArrayList;
+
+@Controller
+public class Controllers {
+    @GetMapping("/hello")
+    public String getGreeting(){
+        return "./resources/templates/hello_world.html";
```

Comment:
You should just be able to write this line as `return "hello_world";`

Thymeleaf automatically knows to look in the templates folder for an html file with that name.